apps/web/scripts/docker/read-secrets.sh

```
@@ -86,22 +86,17 @@ else
-  else
-    echo "✅ Sentry auth token found but skipping upload on ${TARGETARCH:-unknown} platform."
-    echo "ℹ️  Sourcemaps will only be uploaded on amd64 platform to avoid duplicates."
-    echo "🔧 Debug IDs will still be injected for proper error correlation."
-  fi
+  # The token must be exported on every platform, not just amd64. next.config.mjs only wraps
+  # the app in withSentryConfig when SENTRY_AUTH_TOKEN is set, so gating the export by arch
+  # meant an arm64 image got no Debug IDs at all and its stack traces could never be
+  # symbolicated. Uploading the same Debug IDs twice is idempotent and costs a little build
+  # time; shipping an unsymbolicatable image is not a trade worth making.
+  IFS= read -r SENTRY_AUTH_TOKEN < /run/secrets/sentry_auth_token || true
+  SENTRY_AUTH_TOKEN=${SENTRY_AUTH_TOKEN%$'\n'}
```

Shellcheck (0.11.0)

[warning] In POSIX sh, $'..' is undefined.

(SC3003)